README.md

```diff
@@ -68,53 +68,62 @@ You would need to install `apt-transport-https` that supports adding an `https`
 
 here are the steps to perform:
 
-```
-apt-get install -y apt-transport-https
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
-add-apt-repository "deb https://raw.githubusercontent.com/nisabek/icinga2-slack-notifications/master/reprepro general main"
-apt-get update
+```bash
+# apt-get install -y apt-transport-https
+# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
+# add-apt-repository "deb https://raw.githubusercontent.com/nisabek/icinga2-slack-notifications/master/reprepro general main"
+# apt-get update
 ```
 
 You are now ready to install the plugin with 
 
-`apt-get install icinga2-slack-notifications`
+```
+# apt-get install icinga2-slack-notifications
+```
 
 This will create the plugin files in the correct `icinga2` conf directory. 
 
 ### Installation using git
 
-1. clone the repository under your Icinga2 `/etc/icinga2/conf.d` directory
+1. Clone the repository under `/opt` and copy the relevant folder into your Icinga2 `/etc/icinga2/conf.d` directory
  
- `git clone git@github.com:nisabek/icinga2-slack-notifications.git /etc/icinga2/conf.d/`
+```
+# git clone https://github.com/nisabek/icinga2-slack-notifications.git /opt/icinga2-slack-notifications
```

Comment:
well, it's everyones personal choice to do https or ssh...

Reply:
You're completely right, I've had some weird issue where git repo refused to be cloned due to lack of permissions (???) and just switched to https as a result, probably some weird temporary github issue

Reply:
Reverted